### ISSUE TITLE
Styling of code examples in default template don't fulfill accessibility criteria of WCAG AA Level

### ISSUE THREAD

**Issue by Bukama** [asciidoctor/asciidoctor#4381]
The topic came up in an [issue about accessibility problem in the the JUnit user guide](https://github.com/junit-team/junit5/issues/3071), where it was mentioned that JUnit it using the default asciidoc template to create their user guide. 

The default template has a  very low contrast on code examples, which violates the level AA requirement "1.4.3 Contrast (Minimum) (Level AA)" of the WCAGG (which itself is baseline for many accessibility laws, e.g. the German BITV 2.0).

![image](https://user-images.githubusercontent.com/4354359/198874492-2e275c00-cd1a-4808-86bc-b5ba0b0d8775.png)

The code examples have a contrast ratio of 4.49:1, but WCAG level AA requires to meet 4.5:1. So only slight adjustments should fix all of these errors. For example changing the current `#008080` to `#005555` should result in a contrast fulfilling even AAA level.

**Comment by mojavelinux** [comment]
If you're specifically referring to the text in the source blocks, Asciidoctor has no control over the colors that are selected for syntax highlighting. That is decided by the syntax highlighter. You're free to use a different style. For details, see this page: https://docs.asciidoctor.org/asciidoctor/latest/syntax-highlighting/rouge/#rouge-attributes

**Comment by Bukama** [comment]
So asccidoc only used 3rd party syntax highlighter?

**Comment by mojavelinux** [comment]
Asciidoctor only uses third party syntax highlighters, that is correct. This project does not handle syntax highlighting. That would be a very large endeavor and not the focus of this project.

**Comment by Bukama** [comment]
Okay, I close this issue then. Thanks for info
